### DISCUSSION TITLE
Capstone ML

### DISCUSSION THREAD

**Discussion by adrianaj1107** [teamalgoritma/community#152] [Q&A] [answered]
Dear mentor,

![image](https://user-images.githubusercontent.com/104640746/175457646-ceb3b6ee-ec45-483d-a2bc-d0eba74f2770.png)
saya lagi mau buat visualisasi utk seasonality..keluar notif seperti itu..maksudnya apa ya?
![image](https://user-images.githubusercontent.com/104640746/175457706-d21a49d1-bba3-44ef-a6f5-4e1ed3049e0f.png)
![image](https://user-images.githubusercontent.com/104640746/175457713-fb72abe6-680c-4d0f-84aa-d8295e1cf555.png)

Lalu kenapa sumbu Ynya rangenya sampai 1000, padahal di dataframe saya lihat hanya range 20.

Mohon pencerahannya,
terima kasih.
Adriana

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @adrianaj1107 
Melihat dari code Anda, sepertinya diberikan warning bahwa untuk data yang dimasukan di sumbu y (berupa hasil decomposisi raw), tidak dapat ditentukan secara otomatis cara menampilkannya di plot (sumbu y). Akhirnya digunakan cara penampilan default berupa sumbu nilai continuous (angka). Dari code Anda juga, tampaknya nilai pada sumbu y menampilkan nilai seasonal yang digabungkan dari keseluruhan hari. Boleh di cek apakah skala nilai tersebut sesuai plot? Selain itu, sepertinya sebelum melakukan plotting, perlu dilakukan penghitungan rata-rata dulu untuk seasonalnya di tiap periode agar skala nya tepat.

Mengenai cara pembacaannya, nilai seasonal adalah nilai naik/turunnya target kita pada periode tertentu. Bisa ditambahkan/dikurangi sekian poin seasonal dari titik trend/rata-ratanya.

Semoga informasi ini bermanfaat!
